include/ensmallen_bits/sdp/primal_dual_impl.hpp

```diff
@@ -33,47 +33,8 @@
 #include "lin_alg.hpp"
 
 namespace ens {
-
-template<typename DeprecatedSDPType>
-ens_deprecated
-PrimalDualSolver<DeprecatedSDPType>::PrimalDualSolver(
-    const DeprecatedSDPType& sdp) :
-    deprecatedSDP(sdp),
-    initialX(arma::eye<arma::mat>(sdp.N(), sdp.N())),
-    initialYSparse(arma::ones<arma::vec>(sdp.NumSparseConstraints())),
-    initialYDense(arma::ones<arma::vec>(sdp.NumDenseConstraints())),
-    initialZ(arma::eye<arma::mat>(sdp.N(), sdp.N())),
-    maxIterations(1000),
-    tau(0.99),
-    normXzTol(1e-7),
-    primalInfeasTol(1e-7),
-    dualInfeasTol(1e-7)
-{ /* Nothing to do. */ }
-
-template<typename DeprecatedSDPType>
-ens_deprecated
-PrimalDualSolver<DeprecatedSDPType>::PrimalDualSolver(
-    const DeprecatedSDPType& sdp,
-    const arma::mat& initialX,
-    const arma::vec& initialYSparse,
-    const arma::vec& initialYDense,
-    const arma::mat& initialZ) :
-    deprecatedSDP(sdp),
-    initialX(initialX),
-    initialYSparse(initialYSparse),
-    initialYDense(initialYDense),
-    initialZ(initialZ),
-    maxIterations(1000),
-    tau(0.99),
-    normXzTol(1e-7),
-    primalInfeasTol(1e-7),
-    dualInfeasTol(1e-7)
-{
-  // Nothing to do.
-}
-
-template<typename DeprecatedSDPType>
-PrimalDualSolver<DeprecatedSDPType>::PrimalDualSolver(
+inline
+PrimalDualSolver::PrimalDualSolver(
     const size_t maxIterations,
```

Comment:
I think we could fit this on the line above now, in accordance with the style guidelines

Reply:
Ok.  If you agree with me about the `inline` being necessary, I will move the `PrimalDualSolver::PrimalDualSolver` up one line.

Reply:
I think we can fit the one of the parameters too: 
`inline PrimalDualSolver::PrimalDualSolver(const size_t maxIterations,`
This is less than 80 characters